packages/web/src/components/search/SearchBox.js

```diff
@@ -1308,9 +1327,9 @@ const SearchBox = (props) => {
 
 			return sourceObj._id;
 		};
-		return showSourceDocuments
-			&& showAIScreenFooter
-			&& Array.isArray(sourceDocIds) && sourceDocIds.length ? (
+		return (showSourceDocuments
+			&& (showAIScreenFooter)
+			&& Array.isArray(sourceDocIds) && sourceDocIds.length) || props.testMode ? (
```

Comment:
`showSourceDocuments` should be the deciding factor since it's a user controlled prop.

The final condition IMO should be

```js
		return (showSourceDocuments
			&& showAIScreenFooter
			&& Array.isArray(sourceDocIds) && sourceDocIds.length) || (props.testMode &&  showSourceDocuments) ? (
```

Reply:
OK. Got it.